<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiOlbaFlinx</class>
 <widget class="QMainWindow" name="UiOlbaFlinx">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>646</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>930</width>
    <height>646</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayoutCentralwidget">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidgetMain">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pageDataVaults">
       <layout class="QVBoxLayout" name="verticalLayoutDataVaults">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widgetDataVaultsInfo" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>82</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>12</number>
           </property>
           <item>
            <widget class="QLabel" name="labelDataVaultsInfoIcon">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="pixmap">
              <pixmap resource="../../res/olbaflinx.qrc">:/app/olbaflinx-logo</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelDataVaultsInfoTitle">
             <property name="text">
              <string>OlbaFlinx - Online Banking For Linux</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollAreaDataVaults">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaDataVaultsContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>98</width>
             <height>16</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutDataVaults">
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <spacer name="horizontalSpacerDataVaults">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonAddDataVaults">
            <property name="icon">
             <iconset resource="../../res/olbaflinx.qrc">
              <normaloff>:/datavault/add</normaloff>:/datavault/add</iconset>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageApplication">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="olbaflinx::app::banking::AccountWidget" name="treeWidgetAccounts">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>Accounts</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidgetApplication">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="documentMode">
           <bool>false</bool>
          </property>
          <widget class="olbaflinx::app::banking::tabs::TransactionTab" name="tabTransactions">
           <attribute name="title">
            <string>Transactions</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tabStandingOrders">
           <attribute name="title">
            <string>Standing Orders</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tabDocuments">
           <attribute name="title">
            <string>Documents</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tabJobs">
           <attribute name="title">
            <string>Jobs</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>olbaflinx::app::banking::tabs::TransactionTab</class>
   <extends>QWidget</extends>
   <header>TransactionTab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>olbaflinx::app::banking::AccountWidget</class>
   <extends>QTreeWidget</extends>
   <header>AccountWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../res/olbaflinx.qrc"/>
 </resources>
 <connections/>
</ui>
